h/templates/layouts/account.html.jinja2

```diff
@@ -28,13 +28,7 @@
           {% endfor %}
         </ul>
       </nav>
-      {% if request.session.peek_flash('success') -%}
-        <div class="form-flash">
-          {% for message in request.session.pop_flash('success') %}
-            <p>{{ message }}</p>
-          {%- endfor %}
-        </div>
-      {%- endif %}
+      {% include "h:templates/includes/flash-messages.html.jinja2" %}
```

Comment:
This replaces existing flash message code with the new equivalent shared include, so it makes no difference other than removing duplication. I don't _think_ flash messages every do actually appear on `account.html.jinja2` though :man_shrugging: